config/default.schema.json

```diff
@@ -286,6 +286,52 @@
 				}
 			}
 		},
+		"FWU_CONTENT": {
+			"type": "object",
+			"description": "Properties of the S3 storage containing FWU content",
+			"required": [
+				"S3_ENDPOINT",
+				"S3_REGION",
+				"S3_BUCKET",
+				"S3_ACCESS_KEY",
+				"S3_SECRET_KEY"
+			],
+			"properties": {
+				"S3_ENDPOINT": {
+					"type": "string",
+					"format": "uri",
+					"description": "The URI of the S3 provider"
+				},
+				"S3_REGION": {
+					"type": "string",
+					"default": "de",
+					"description": "Region name of bucket"
+				},
+				"S3_BUCKET": {
+					"type": "string",
+					"description": "Bucket name"
+				},
+				"S3_ACCESS_KEY": {
+					"type": "string",
+					"description": "Access Key to S3"
+				},
+				"S3_SECRET_KEY": {
+					"type": "string",
+					"description": "Secret key to S3"
+				},
+				"INCOMING_REQUEST_TIMEOUT": {
+					"type": "integer",
+					"minimum": 0,
+					"default": 6000000,
```

Comment:
On some places we have 60 sec, but this only for long copy processes and we can set what we want, after 30 sec the request is killed from our infastructure. Please set it to a realistic time.

Reply:
At 60000 we get timeouts under 60 seconds.
Maybe it's not ms.